Source/Libs/CairoSharp/Context.cs

```diff
@@ -65,7 +65,7 @@ static Context ()
 			// except in the case of Win64 where sizeof(long)
 			// is 32 bits
 			//
-			int ptr_size = Marshal.SizeOf (typeof (IntPtr));
+			int ptr_size = Marshal.SizeOf<IntPtr>();
```

Comment:
Marshal.SizeOf is unnecessary overhead for blittable types. It is better to just use `sizeof(IntPtr)`, sizeof(ulong)`, etc.